schema/iguana.owl

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
I think in general it would make sense to use the Protégé generated file hear instead of the manually written one. This is RDF/XML if I am not mistaken (so not OWL/XML).